calculator-app/src/main/java/camp/nextstep/edu/calculator/viewmodel/CalculatorViewModel.kt

```diff
@@ -3,12 +3,21 @@ package camp.nextstep.edu.calculator.viewmodel
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
 import camp.nextstep.edu.calculator.SingleLiveEvent
 import camp.nextstep.edu.calculator.domain.Calculator
 import camp.nextstep.edu.calculator.domain.Expression
 import camp.nextstep.edu.calculator.domain.Operator
+import camp.nextstep.edu.calculator.domain.repo.History
+import camp.nextstep.edu.calculator.domain.repo.HistoryRepository
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class CalculatorViewModel(
+    expression: Expression? = null,
```

Comment:
이전에 만들어 두었던 빈 수식을 디폴트로 넣어주는 건 어떨까요?
```suggestion
class CalculatorViewModel(
    expression: Expression = Expression.EMPTY
```

Reply:
수정해보았습니다.